src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,50 @@
+import React from "react";
+import { getNumbers } from "../../utils";
+
+interface Props {
+  buttonsLength: number,
+  activePage: number,
+  callback: (page:number) => void,
+}
+
+export const Pagination : React.FC<Props> = ({buttonsLength, activePage, callback}) => {
+  const buttonsItems = getNumbers(1, buttonsLength);
+
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${activePage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          onClick={() => callback(activePage === 1 ? 1 : activePage - 1)}
+          aria-disabled={activePage === 1}>
+          «
+        </a>
+      </li>
+      {buttonsItems.map(n => (
+        <li  key={n} className={`page-item ${activePage === n ? 'active' : ''}`}>
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={'#' + n}
```

Comment:
Using `href={'#' + n}` for page links might not be ideal for accessibility. Consider using a `button` element instead of an `a` tag for actions that don't navigate to a new page.